pkg/blob/controllerserver.go

```diff
@@ -667,6 +682,107 @@ func (d *Driver) DeleteBlobContainer(ctx context.Context, subsID, resourceGroupN
 	})
 }
 
+// CopyBlobContainer copies a blob container in the same storage account
+func (d *Driver) CopyBlobContainer(ctx context.Context, subsID, resourceGroupName, accountName, accountKey, srcContainerName, dstContainerName string, secrets map[string]string) error {
+	if srcContainerName == "" {
+		return fmt.Errorf("srcContainerName is empty")
+	}
+	if dstContainerName == "" {
+		return fmt.Errorf("dstContainerName is empty")
+	}
+	return wait.ExponentialBackoff(d.cloud.RequestBackoff(), func() (bool, error) {
+		var err error
+		if acquired := d.volumeLocks.TryAcquire(dstContainerName); !acquired {
+			return true, status.Errorf(codes.Aborted, volumeOperationAlreadyExistsFmt, dstContainerName)
+		}
+		defer d.volumeLocks.Release(dstContainerName)
+		if len(secrets) > 0 {
+			accountSasToken, rerr := getAccountSasToken(secrets, accountName)
+			if rerr != nil || accountSasToken == "" {
+				klog.V(2).Infof("get account sas token from secrets err: %s, sasToken: %s", err.Error(), accountSasToken)
```

Comment:
don't print two lines of logs one by one, combine as one log

Reply:
done